test/e2e/common/utils.py

```diff
@@ -13,39 +13,46 @@
 # limitations under the License.
 import time
 import logging
+import json
+import requests
+from urllib.parse import urlparse
 from kubernetes import client, config
 from kfserving import KFServingClient
 
 logging.basicConfig(level=logging.INFO)
 
 KFServing = KFServingClient(config_file="~/.kube/config")
+KFSERVING_NAMESPACE = "kfserving-system"
+KFSERVING_TEST_NAMESPACE = "kfserving-ci-e2e-test"
 
-def wait_for_isvc_ready(name, namespace='kfserving-ci-e2e-test',
-                             Timeout_seconds=600, debug=True):
-    for _ in range(round(Timeout_seconds/10)):
+
+def wait_for_isvc_ready(name, namespace=KFSERVING_TEST_NAMESPACE,
+                        timeout_seconds=600, debug=True):
+    for _ in range(round(timeout_seconds/10)):
         time.sleep(10)
         kfsvc_status = KFServing.get(name, namespace=namespace)
         status = 'Unknown'
         for condition in kfsvc_status['status'].get('conditions', {}):
             if condition.get('type', '') == 'Ready':
                 status = condition.get('status', 'Unknown')
         if status == 'True':
+            time.sleep(10)
             return
     if debug is True:
         logging.warning("Timeout to start the InferenceService %s.", name)
         logging.info("Getting the detailed InferenceService ...")
         logging.info(KFServing.get(name, namespace=namespace))
         get_pod_log(pod='kfserving-controller-manager-0',
-                    namespace='kfserving-system',
+                    namespace=KFSERVING_NAMESPACE,
```

Comment:
We're going to get into trouble when users deploy this to the kubeflow namespace

Reply:
Should be ok for our tests though.